src/data/ACTIONS/WAR.js

```diff
@@ -8,27 +8,31 @@ export default {
 		name: 'Steel Cyclone',
 		icon: 'https://xivapi.com/i/002000/002552.png',
 		onGcd: true,
+		breaksCombo: false,
```

Comment:
By no means a blocker as it doesn't hurt anything, but specifying this as `false` isn't strictly necessary.

Reply:
I did it for completeness sake. If/when something that checks that ends up happening, it's there. I know it's mostly unnecessary, but figured I'd do it anyway.

Reply:
I meant it more in a sense of "missing property is falsey", but that's fair. If we do explicit checks it'll be good to have.

Reply:
Oh no, I understood. I just don't think it's a big deal to have (or to not have it), toastyboy.